hack/lib/golang.sh

```diff
@@ -497,6 +497,15 @@ EOF
 #   env-var GO15VENDOREXPERIMENT=1
 #   current directory is within GOPATH
 kube::golang::setup_env() {
+  # default GO_VERSION to content of .go-version
+  GO_VERSION="${GO_VERSION:-"$(cat "${KUBE_ROOT}/.go-version")"}"
```

Comment:
should this be a local variable?

Reply:
ah, ok, you allow to override it

Reply:
yeah, probably, will clean this up when we're happy with the overall shape